src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
+    </h2>
+
+    {person.age ? (
+      <p className="Person__age">
+        I am
+        { ' ' }
+        {person.age}
+      </p>
```

Comment:
```suggestion
      <p className="Person__age">
        {`I am ${person.age}`}
      </p>
```
we can simplify like this

Reply:
let's fix it everywhere